pkg/cmd/cli/cmd/rollback.go

```diff
@@ -221,12 +221,13 @@ func (o *RollbackOptions) Run() error {
 
 	// If this is a dry run, print and exit.
 	if o.DryRun {
-		describer := describe.NewDeploymentConfigDescriberForConfig(o.oc, o.kc, newConfig)
-		description, err := describer.Describe(newConfig.Namespace, newConfig.Name)
+		// Compare against the current deployment config
+		configName := deployutil.DeploymentConfigNameFor(target)
+		currentConfig, err := o.oc.DeploymentConfigs(o.Namespace).Get(configName)
 		if err != nil {
 			return err
 		}
-		o.out.Write([]byte(description))
+		o.out.Write([]byte(kutil.ObjectGoPrintSideBySide(currentConfig, newConfig)))
```

Comment:
It will dump all the dc config. Unfortunately I cannot find anything
similar for descriptions

On Fri, Oct 16, 2015 at 2:57 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/rollback.go
> <https://github.com/openshift/origin/pull/5157#discussion_r42238842>:
>
> >  		if err != nil {
> >  			return err
> >  		}
> > -		o.out.Write([]byte(description))
> > +		o.out.Write([]byte(kutil.ObjectGoPrintSideBySide(currentConfig, newConfig)))
>
> Won't this print out go struct field names, which won't correspond to JSON
> fields the user knows about?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5157/files#r42238842>.
>